tetrateci/cleanup_gke_cluster.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+SHA8=$(git rev-parse --short $GITHUB_SHA)
+CLUSTER_NAME="test-istio-$SHA8"
```

Comment:
add version to the cluster name - otherwise can't spin multiple versions

Reply:
ahh right